packages/ui/react-ui-editor/src/components/TextEditor/theme.ts

```diff
@@ -19,15 +20,16 @@ export const defaultStyles: {
   '&.cm-focused': {
     outline: 'none',
   },
-  '.cm-placeholder': {
-    fontFamily: get(tokens, 'fontFamily.body', []).join(','),
-  },
-  '& .cm-scroller': {
+  '.cm-scroller': {
     fontFamily: get(tokens, 'fontFamily.body', []).join(','),
     overflow: 'visible',
   },
   '.cm-tooltip': {
     backgroundColor: 'transparent',
     border: 'none',
   },
+  '.cm-link': {
+    color: get(tokens, 'colors.primary.500'), // TODO(burdon): Not working.
```

Comment:
`tokens` is really just our extension of the default tailwind theme, so to get color tokens it’s `get(tokens, 'extend.colors.primary.500')`.